ports/servoshell/desktop/headed_window.rs

```diff
@@ -646,9 +664,6 @@ impl WindowPortsMethods for Window {
                     winit::window::Theme::Dark => Theme::Dark,
                 });
             },
-            WindowEvent::Moved(_new_position) => {
-                webview.notify_embedder_window_moved();
```

Comment:
What is the rationale for removing this callback?

Reply:
It's no longer necessary as the compositor no longer caches `EmbedderCoordinates` and always calls into the delegate now, so there isn't any need to notify it that the window moved.